src/amo/components/UserProfileEdit/index.js

```diff
@@ -610,9 +655,79 @@ export class UserProfileEditBase extends React.Component<Props, State> {
                     i18n.gettext("Update user's profile")
                 )}
               </Button>
+              <Button
+                buttonType="neutral"
+                className="UserProfileEdit-button UserProfileEdit-delete-button"
+                disabled={!user}
+                onClick={this.onDeleteProfile}
+                puffy
+                type="button"
+              >
+                {isEditingCurrentUser ?
+                    i18n.gettext('Delete my profile') :
+                    i18n.gettext(`Delete user's profile`)}
+              </Button>
             </div>
           </div>
         </form>
+
+        {this.state.showProfileDeletionModal && (
+          <OverlayCard
+            className="UserProfileEdit-deletion-modal"
+            header={isEditingCurrentUser ? i18n.gettext(
+              `Attention: you are about to delete your profile. Are you sure?`
+            ) : i18n.gettext(
+              `Attention: you are about to delete a profile. Are you sure?`
+            )}
+            onEscapeOverlay={this.onCancelProfileDeletion}
+            visibleOnLoad
+          >
+            <p
+              // eslint-disable-next-line react/no-danger
+              dangerouslySetInnerHTML={
+                sanitizeHTML(i18n.sprintf(
```

Comment:
I guess we need this in case a translator tries to sneak some bad html into a translation?

Reply:
Indeed, and that's what we do everywhere when we need to insert HTML tags in translations.

Reply:
Indeed! We use this pattern for translation strings including HTML (links most of the time).